src/ripple/shamap/tests/SHAMapSync.test.cpp

```diff
@@ -43,7 +43,8 @@ class sync_test : public beast::unit_test::suite
 
         for (int d = 0; d < 3; ++d) s.add32 (rand ());
 
-        return std::make_shared<SHAMapItem> (to256 (s.getRIPEMD160 ()), s.peekData ());
+        return std::make_shared<SHAMapItem>(
+            s.getSHA512Half(), s.peekData ());
```

Comment:
I'm not sure what's going on here with the hash function being changed. That should change the data, but it doesn't seem to hurt the test.